my-website/docs/cloneCozie.md

```diff
@@ -10,4 +10,4 @@ In order to edit the source code and being able to work on Cozie you will need t
 
 Please follow the instructions in the video below.
 
-<iframe width="100%" height="315" src="https://www.youtube.com/embed/gSNPvoGc8Zw" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
Are you sure about this code change? When I try to share an iframe from YouTube, `allowfullsreen` is all lowercase. See image below.

![image](https://user-images.githubusercontent.com/40018640/112920850-5bd9d580-913c-11eb-9869-da85a2b127f8.png)